scripts/rosa/rosa.sh

```diff
@@ -260,11 +288,11 @@ main() {
     while :; do
         case "${1:-}" in
         provision_sts_cluster)
-            provision_sts_cluster
+            provision_rosa_cluster
             exit 0
             ;;
         delete_sts_cluster)
-            delete_sts_cluster
+            delete_rosa_cluster
             exit 0
             ;;
         rhoam-prerequisites)
```

Comment:
```suggestion
        rhoam_prerequisites)
```
Need to change this line and the line below (it wouldn't let me comment on that line) to `rhoam_prerequisites` because its [entrypoint](https://github.com/integr8ly/delorean/pull/284/files#diff-24835738dda67c8b67bfbc5ce87276ef690466eb2da99d467504a09ee1760057R88) in the makefile was changed to use an `_`